tox.ini

```diff
@@ -23,17 +23,22 @@ match-dir = (?!migrations)
 
 [pytest]
 DJANGO_SETTINGS_MODULE = test_settings
-addopts = --cov enterprise --cov integrated_channels --cov-report term-missing --cov-report xml
+addopts = --cov enterprise --cov consent --cov integrated_channels --cov-report term-missing --cov-report xml
 norecursedirs = .* docs requirements
 
 [testenv]
+whitelist_externals =
+    rm
+    touch
 deps =
     ficus: -r{toxinidir}/requirements/test-ficus.txt
     ginkgo: -r{toxinidir}/requirements/test-ginkgo.txt
     hawthorn: -r{toxinidir}/requirements/test-hawthorn.txt
     master: -r{toxinidir}/requirements/test-master.txt
 commands =
+    touch tests/__init__.py
     py.test -Wd {posargs}
+    rm tests/__init__.py
```

Comment:
Hmm, OK so I read this carefully:

```
setup.py
mypkg/
    ...
tests/
    __init__.py
    foo/
        __init__.py
        test_view.py
    bar/
        __init__.py
        test_view.py

Now pytest will load the modules as tests.foo.test_view and tests.bar.test_view, allowing you to have modules with the same name. But now this introduces a subtle problem: in order to load the test modules from the tests directory, pytest prepends the root of the repository to sys.path, which adds the side-effect that now mypkg is also importable. This is problematic if you are using a tool like tox to test your package in a virtual environment, because you want to test the installed version of your package, not the local code from the repository.

In this situation, it is strongly suggested to use a src layout where application root package resides in a sub-directory of your root:

setup.py
src/
    mypkg/
        __init__.py
        app.py
        view.py
tests/
    __init__.py
    foo/
        __init__.py
        test_view.py
    bar/
        __init__.py
        test_view.py

This layout prevents a lot of common pitfalls and has many benefits, which are better explained in this excellent blog post by Ionel Cristian Mărieș.
```

And for that reason, since I don't think I should be making a new `src/` directory (or some other appropriate name) as part of this PR, I'm moving `utils` back out to root as `test_utils` to avoid the `__init__.py`. In a future PR we can solve this problem.